### PR TITLE
Log level console arg

### PR DESCRIPTION
**Pull request by DrDab**
Made log level specifiable using command line argument.

### CODE REVIEW

File: src/Globals.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #include "Globals.h"
-
 #include "Constants.h"
+#include "log.h"
```
Comment (abhaybd) [inline src/Globals.cpp:3]: Is this include necessary?

File: src/log.cpp [hunk 2]
```diff
@@ -1,17 +1,21 @@
-
+#include "Globals.h"
```
Comment (abhaybd) [inline src/log.cpp:1]: Do we need this include either?

File: src/log.h [hunk 3]
```diff
@@ -3,12 +3,11 @@
...
-extern int LOG_LEVEL;
-
 constexpr int LOG_TRACE = 0;
 constexpr int LOG_DEBUG = 1;
 constexpr int LOG_INFO = 2;
 constexpr int LOG_WARN = 3;
 constexpr int LOG_ERROR = 4;
 
 void log(int level, const char* fmt, ...);
+void setLogLevel(int level);
```
Comment (abhaybd) [inline src/log.h:13]: I know it wasn't there before, but could we add doxygen documentation to these methods?

**Review by abhaybd — CHANGES_REQUESTED** [review]
Overall well done, just some minor nitpicks

**Comment by DrDab** [comment]
Added documentation for log function and removed the unnecessary includes.

**Review by abhaybd — APPROVED** [review]
Nice clean code, LGTM!

**Pull request merged**